### PR TITLE
Almost complete user sign-up, add Figaro and sendgrid, html, css - submit button not working

### PR DESCRIPTION
**Pull request by PamBWillenz**
User could sign in before I changed html and css. Now the submit button isn't working.

### CODE REVIEW

**Comment by bjblock** [comment]
which submit button?

**Comment by PamBWillenz** [comment]
Sign In submit button - after you click first sign in button (blue) then you get the button and fields all the way to the left - that sign in submit button

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,31 @@
-<h2>Log in</h2>
+<h2>Sign in</h2>
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
+<div class="row">
+  <div class="col-md-8>"
```
Comment (bjblock) [inline app/views/devise/sessions/new.html.erb:4]: typo here is causing the error

Comment (PamBWillenz) [inline app/views/devise/sessions/new.html.erb:4]: thanks. Will fix before our meeting tomorrow @ 9am (EDT).

**Pull request merged**